README.md

```diff
@@ -16,6 +16,8 @@
    ```sh
    cd fxa-local-dev
    npm install
+   npm run start-remote
```

Comment:
this is not correct, this command only works from the `fxa-content-server` repo: https://github.com/mozilla/fxa-content-server#quick-start